pkgs/development/interpreters/python/mk-python-derivation.nix

```diff
@@ -157,6 +157,19 @@ let
   ] ++ checkInputs;
 
   postFixup = lib.optionalString (!dontWrapPythonPrograms) ''
+    # Propagate wrapper arguments up to the python
```

Comment:
In `mk-python-derivation` a hook needs to do this; the function itself should remain small.

@worldofpeace @jtojnar any common approach that can be taken here? I want declarative wrappers in JSON with structured attributes, but we're not there. Let's discuss this in the main thread.

Reply:
If the various hook were to export their arguments in a common array, be it `makeWrapperArgs` or something else it would be great.

Reply:
Ok, I moved the shell code to a dedicated hook. I tested again with matplotlib and it seems good. I used a clone of `buildPythonPackage` to avoid a mass rebuild, so I haven't tested much else.